stdlib/format.c

```diff
@@ -239,3 +239,33 @@ uint256dec(char *output, uint256_t *val256)
 
     return output;
 }
+
+static const char b58digits[] = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
+
+// https://github.com/bitcoin/libbase58/blob/b1dd03fa8d1be4be076bb6152325c6b5cf64f678/base58.c inspired this code.
+void base58_encode(uint8_t * data, uint32_t data_len, uint8_t * output, uint32_t output_len)
```

Comment:
Would be great if this function could be tested directly. But I guess we lack the testing infrastructure for our stdlib

Reply:
There is another PR for this